client/webserver/locales/en-us.go

```diff
@@ -238,4 +238,9 @@ var EnUS = map[string]string{
 	"successful_cert_update":      "Successfully updated certificate!",
 	"update dex host":             "Update DEX Host",
 	"copied":                      "Copied!",
+	"estimated_transaction_fee":   "Estimated Transaction Fee",
+	"estimated_total_spend":       "Estimated Total Spend",
+	"estimated_balance":           "Estimated Balance",
```

Comment:
maybe `Estimated Balance` -> `Estimated Balance After` would be slightly sharper